accepted/global-json-updates.md

```diff
@@ -0,0 +1,790 @@
+    * `disable`
+      * Do not roll forward. Only bind to specified version. This policy means the latest patches, including security patches will not be used.
+    * Defaults: 
+      * If a version is specified in the `global.json`, the default is `patch` to match today's behavior.
+      * If there is no `global.json` or a version is not specified, the default is`latestMajor`, again to match today's behavior.
+ 
+### Examples to illustrate roll-forward options
+
+The following table considers various SDK version combinations that may be available on the machine, and the outcome of the following `global.json`:
+
+```json
+{
+  "sdk": {
+    "version": "2.1.501",
+    "rollForward": [entry in table header]
+  }
+}
+```
+
+| Available SDKs                                       | patch     | feature | minor   | major   | latestPatch | latestFeature | latestMinor | latestMajor | disable |
+|------------------------------------------------------|-----------|---------|---------|---------|-------------|---------------|-------------|-------------|---------|
+| 2.1.500                                              | fail      | fail    | fail    | fail    | fail        | fail          | fail        | fail        | fail    |
+| 2.1.501, 2.1.503                                     | _2.1.501_ | 2.1.501 | 2.1.501 | 2.1.501 | _2.1.503_   | 2.1.503       | 2.1.503     | 2.1.503     | 2.1.501 |
+| 2.1.503, 2.1.505, 2.1.601, 2.2.101, 3.0.100          | 2.1.503   | 2.1.503 | 2.1.503 | 2.1.503 | 2.1.505     | 2.1.601       | 2.2.101     | 3.0.100     | fail    |
+| 2.1.601, 2.1.604, 2.1.702, 2.2.101, 2.2.203, 3.0.100 | fail      | 2.1.601 | 2.1.601 | 2.1.601 | fail        | 2.1.702       | 2.2.203     | 3.0.100     | fail    |
+| 2.2.101, 2.2.203, 3.0.100                            | fail      | fail    | 2.2.101 | 2.2.101 | fail        | fail          | 2.2.203     | 3.0.100     | fail    |
+| 3.0.100, 3.1.102                                     | fail      | fail    | fail    | 3.0.100 | fail        | fail          | fail        | 3.1.102     | fail    |
+
+Or, if it is easier to understand with logic (bail out of logic on fail or select):
+
+1. If not `latest`, and there is a match, select it.
+1. If we get here for `disable`, then fail.
+1. Exclude any lower versions.
+1. If `minor` or `latestMinor`, exclude any that don't match major version. 
+1. If `feature` or `latestFeature`, exclude any that don't match major and minor version.
+1. If `patch` or `latestPatch`, exclude any that don't match major, minor and feature band version.
+1. If there is one left, select it.
+1. If any of the `latest` options are used, select the highest. 
+1. Otherwise, select the lowest major, and within that lowest minor, and within that lowest feature band. Then select the highest patch in that feature band.  
+
+## Customer problems addressed
```

Comment:
I have a scenario where there is a global.json at the root of the repo and I don't want to align with it, but I also don't want to specify anything. I just want the default behavior. What should I do in that case?